pkg/common/common.go

```diff
@@ -39,6 +39,10 @@ const (
 	CLONER_IMAGE_PATH     = "/tmp/clone/image"
 	CLONER_SOCKET_PATH    = "/tmp/clone/socket"
 
+	UPLOAD_SERVER_PODNAME       = "uploadserver"
```

Comment:
Couple of things, we sure do like to set constants and I'm not sure we need them in all cases.  That being said; it would be nice to follow some best practices on Names like s/UPLOAD_SERVER_PODNAME/UploadServerPodName/

Even better, that name seems a bit of a keyboardful.  Also, one thing about all these constants I'm noticing, they're almost always public, and they pretty much never have a comment let alone a godoc friendly comment.

It'd be nice if we were more judicious in our use of Constants, and especially public constants, and if we were to document things a bit more

Reply:
Yeah, I am not a fan of the all caps names here.  Was just following convention.  Totally on board with refactoring constant usage.  But prob best for a separate pr.